meetings/2025-05-01.md

```diff
@@ -0,0 +1,123 @@
+# Minutes of the 6th meeting of Ecma TC55
+
+|                   |                              |
+|:------------------|-----------------------------:|
+| held in:          |        Google Meet (virtual) |
+| on:               |                  01 May 2025 |
+| Chairs:           |  Luca Casonato (Deno), Andreu Botella (Igalia)|
+| Attendees:        | Luca Casonato (Deno), Andreu Botella (Igalia), Ethan Arrowood (HarperDB, Invited Expert), Dean Srebnik (Andromeda, Invited Expert), Owen Buckley (Project Evergreen, Invited Expert), Sam Gammon (Elide Technologies, Invited Expert), Chris de Almeida (Open JSF), Lino Le Van (Chic, Invited Expert), Aki Braun (Ecma International)|
```

Comment:
```suggestion
| Attendees:        | Luca Casonato (Deno), Andreu Botella (Igalia), Ethan Arrowood (HarperDB, Invited Expert), Owen Buckley (Project Evergreen, Invited Expert), Sam Gammon (Elide Technologies, Invited Expert), Chris de Almeida (Open JSF), Aki Braun (Ecma International)|
```

@lino-levan @load1n9 I don't think you joined this meeting, right?

Reply:
Yep.